example/macos/AsyncStorageExample-macOS/RNCTestAsyncStorageDelegate.m

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#import "RNCTestAsyncStorageDelegate.h"
```

Comment:
Can we avoid duplicating `example/ios/AsyncStorageExample/RNCTestAsyncStorageDelegate.m`?

Reply:
yes